iib/web/errors.py

```diff
@@ -2,11 +2,12 @@
 from typing import List, NoReturn
 
 import kombu.exceptions
-from flask import current_app, jsonify, Request, Response
+from flask import current_app, jsonify, Response
 from werkzeug.exceptions import HTTPException
 
 from iib.exceptions import IIBError, ValidationError
 from iib.web import messaging, db
+from iib.web.models import Request
```

Comment:
Based on line 50: `request.add_state('failed', 'The scheduling of the request failed')`, I suppose that we want to use class Request from iib.web.models which inherit from flask.Request. (It is possible that I introduced this import when I was adding the mypy types to the iib).